OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -748,7 +748,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <material key="leather"/>     
                                 </decoNote>
                                 <decoNote xml:id="b3" type="Cover" color="reddish-brown">
-                                    The leather cover is wrapped in a protective flower-printed cotton cover and is decorated  with blind tooled ornament
+                                    The leather cover is wrapped in a protective flower-printed cotton cover and is decorated with blind tooled ornament
                                     (circles, double circles, straight lines, a cross). The ornamentation decorates both the inner and outer sides of the cover.
                                     <!-- Description of the tooled ornamentation might need some revision --></decoNote>
```

Comment:
I am going to systematically check the binding descriptions after the set-up of a selected vocabulary for decorative motifs